po/de.po

```diff
@@ -1542,135 +1626,140 @@ msgid ""
 "\n"
 "And {missing_titles_count:d} more tasks"
 msgstr ""
+"\n"
+"Und {missing_titles_count:d} weitere Aufgaben"
 
 #. These lines should be in info.py, but due to their dynamic nature
 #. there'd be no way to show them translated in Gtk's About dialog:
-#: GTG/gtk/browser/main_window.py:303
+#: GTG/gtk/browser/main_window.py:304
 #, python-format
 msgid "Copyright © 2008-%d the GTG contributors."
-msgstr ""
+msgstr "Copyright © 2008-%d Beitragende von GTG."
 
+# TODO: Sounds akward
 #. GTK prefixes the first line with "Created by ",
 #. but we can't split the string because it would cause trouble for some languages.
-#: GTG/gtk/browser/main_window.py:308
+#: GTG/gtk/browser/main_window.py:309
 msgid "GTG was made by many contributors around the world."
-msgstr ""
+msgstr "GTG wurde von vielen Beitragende weltweit mitgewirkt."
```

Comment:
This sentence is wrong and sounds weird.

Reply:
I wanted to add the "contributors" and "around the world", but I didn't really find a good way. I'll change it to "GTG wurde von vielen beigetragen." if that is OK. (I am unsure if I can insert "weltweit" as the second to last word?)

Reply:
"GTG wurde von vielen weltweit beigetragen" sounds ok to me, although still a bit weird. However, I can't think of a better translation either.

Reply:
Well then lets use that translation for now.